package.json

```diff
@@ -42,7 +42,8 @@
     "commander": "^2.9.0",
     "express": "^4.0.0",
     "mongodb": "*",
-    "semver": "^7.3.4"
+    "semver": "^7.3.4",
+    "fastify-static": "^4.5.0"
```

Comment:
Please remove fastify from prod deps. I know that express.js is here. But all the other frameworks are in dev deps.

(Some day we would need to remove express.js from prod deps too)

Reply:
I fixed it myself. No worries.